proto/rill/runtime/v1/queries.proto

```diff
@@ -276,6 +284,25 @@ message ExportResponse {
   string download_url_path = 1;
 }
 
+message ExportReportRequest {
+  string instance_id = 1;
+  string name = 2;
```

Comment:
nit: maybe named it `string report` instead? We don't really have a convention around this so it's subjective